dxr/static_unhashed/js/dxr.js

```diff
@@ -385,6 +385,8 @@ $(function() {
         if (!data.results.length) {
             resultsLineCount = 0;
             if (!append) {
+                var renderedNav = nunjucks.render('results_nav.html', data);
```

Comment:
These changes in dxr.js are for setting up the nav bar in search results pages, which will show the number of results with a link to the tree's root, and the Switch Tree menu. This is based on some of the discussion in the bugzilla page: https://bugzilla.mozilla.org/show_bug.cgi?id=1288487#c4. I know it's a bit repetitive, but I put those lines in each place where the search results content was being updated, which in turn would affect the "# of results from the code tree" heading.

I'm not sure if it's needed in all three places, so I can take a look into it some more. I think at least one of them can be removed, in the infinite scroll case.

Just for reference, here is a screenshot of it:
![search_dxr](https://cloud.githubusercontent.com/assets/2381807/19175789/d9879772-8c07-11e6-8d25-1ae0145682ce.jpg)